content/kubermatic/main/tutorials-howtos/gitops-argocd/_index.en.md

```diff
@@ -0,0 +1,241 @@
+
+This is why, if we utilize a GitOps solution to manage KKP and it's upgrades, KKP administrator would have better peace of mind. We have now provided [alpha release of ArgoCD based management of KKP master and seeds](https://github.com/kubermatic/kubermatic/tree/main/charts/gitops/kkp-argocd-apps).
+
+This page outlines how to install ArgoCD and team provided Apps to manage KKP seeds and their upgrades.
+
+## Preparation
+
+In order to setup and manage KKP using a GitOps solution, one must first know some basics about KKP. Following links could be useful to get that knowledge, if you are new to KKP.
+
+1. [KKP main documentation home](../../) For general overview. This documentation is quite vast and you should glance through this documentation but focus on specific links below.
+1. [KKP Architecture, terminology and planning](../../architecture/)
+1. [Hardware requirements, firewall requirements](../../architecture/requirements/cluster-requirements/) and [supported cloud providers](../../architecture/supported-providers/) and [DNS requirements](../../installation/install-kkp-ce/#update-dns--tls)
+
+We will install KKP along the way, so, we do not need a running KKP installation. But if you already have KKP installation running, you can still make use of this guide to onboard your existing KKP installation to ArgoCD. This might involve in directory re-organization, though.
+
+## Introduction
+
+Below diagram shows the general concept of the setup. It shows how KKP installations and ArgoCD will be deployed and what KKP components will be managed by ArgoCD in each seed.
+![Concept](@/images/tutorials/gitops-argocd/kkp-gitops-argocd.png "Concept - KKP GitOps using ArgoCD")
+
+For the demonstration, 
+1. We will use 2 kubernetes clusters in AWS (created using Kubeone but they could be any Kubernetes clusters on any of the supported cloud / on-prem providers as long as they have a network path to reach each other)
+1. Install KKP master on one cluster (c1) and also use this cluster as seed (master-seed combo cluster)
+1. Make 2nd cluster (c2) as dedicated seed
+1. ArgoCD will be installed in each of the seed (and master/seed) to manage the respective seed's KKP components
+
+A high-level procedure to get ArgoCD to manage the seed would be as follows:
+1. Setup a kubernetes cluster to be used as master (or master-seed combo)
+1. Install ArgoCD helm chart and KKP ArgoCD Applications helm chart
+1. Install KKP on the master seed using kubermatic-installer
+1. Setup DNS records and Git repository tag so that ArgoCD can sync
+1. Sync the applications
+1. Repeat the procedure for 2nd seed (except installation of KKP on the seed)
```

Comment:
* for _the_ 2nd seed
* "Repeat the procedure for 2nd seed (except installation of KKP on the seed)" -- I don't get it. Repeat it, but don't do anything?